seagence/manifest.json

```diff
@@ -0,0 +1,69 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "94f4e504-c98c-466f-b934-5e5ee0331944",
+  "app_id": "seagence",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Defect Monitoring and Root Cause Automation using Seagence",
+    "title": "Seagence",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Defect and Success Clusters",
+        "image_url": "images/defect-and-success-clusters.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Success and Defect Execution Paths",
+        "image_url": "images/defect-and-successexecution-paths-1440x810.png"
+      },
+      {
+        "media_type": "image",
```

Comment:
Can we move this image so that it appears first?

Reply:
@srinivas-bitla Looks like this is still unresolved - to make this image appear first, please change the order of the media objects so that this one is first in the array

Reply:
To be clear, the request is that the image with the dashboard appears first